devito/ops/compiler.py

```diff
@@ -48,8 +44,11 @@ def jit_compile(self, soname, ccode, hcode):
                 self.code = f.read()
         except FileNotFoundError:
             warning("Couldn't find file: %s" % self.ops_src)
-        else:
-            self._compile_cuda(soname)
+
+        if self._devito_platform == 'nvidiaX':
```

Comment:
... here, you should just look at `configuration['platform']`. No need for `_devito_platform`

'seq' should be replaced by 'intel64' or something along these lines (see [here](https://github.com/opesci/devito/blob/master/devito/archinfo.py#L249) )

Reply:
ok, fixing